### DISCUSSION TITLE
Question:

### DISCUSSION THREAD

**Discussion by SinonsAnimations** [FunkinCrew/Funkin#1552] [Compiling help]
i have dis error /HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in 
favor of haxe.xml.Access  What I can do?

**Comment by libauror-a** [comment]
this shoud be a warning instead of a error

**Comment by NotHyper-474** [comment]
Warnings, not errors. They do not stop compilation but they tell something might be wrong with the code. In this case it's a problem with Polymod so you really can't do much about it.

**Comment by ghost** [comment]
> sussy mogus

?

**Comment by ghost** [comment]
> > sussy mogus
> 
> ?

According to @aflacc, sus

**Comment by ghost** [comment]
> > > sussy mogus
> 
> > 
> 
> > ?
> 
> 
> 
> According to @aflacc, sus

sus

**Comment by aflacc** [comment]
> > > > sussy mogus
> > 
> > 
> > > 
> > 
> > 
> > > ?
> > 
> > 
> > According to @aflacc, sus
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > sussy mogus
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > ?
> 
> > > 
> 
> > > 
> 
> > > According to @aflacc, sus
> 
> > 
> 
> > sus
> 
> 
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > > sussy mogus
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > >
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > > ?
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > According to @aflacc, sus
> > 
> > 
> > > 
> > 
> > 
> > > sus
> > 
> > 
> > sus
> 
> sus

sus

**Comment by aflacc** [comment]
> > > > > > > sussy mogus
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > > ?
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > According to @aflacc, sus
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > sus
> > > 
> > > 
> > > sus
> > 
> > 
> > sus
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > > > > sussy mogus
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > > ?
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > >
> > > > 
> > > > 
> > > > > > According to @aflacc, sus
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > sus
> > > > 
> > > > 
> > > > sus
> > > 
> > > 
> > > sus
> > 
> > 
> > sus
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > > > > > sussy mogus
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > > ?
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > >
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > > According to @aflacc, sus
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > 
> 
> > > > > 
> 
> > > > > 
> 
> > > > > > sus
> 
> > > > > 
> 
> > > > > 
> 
> > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > sus
> 
> > > 
> 
> > > 
> 
> > > sus
> 
> > 
> 
> > sus
> 
> 
> 
> sus

sus

**Comment by libauror-a** [comment]
> > > > > > > > > > sussy mogus
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > > > ?
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > > According to @aflacc, sus
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > > sus
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > sus
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > sus
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > sus
> > 
> > 
> > > 
> > 
> > 
> > > sus
> > 
> > 
> > sus
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > > > > > > > sussy mogus
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > > > ?
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > > According to @aflacc, sus
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > > sus
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > sus
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > > > sus
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > sus
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > sus
> > > 
> > > 
> > > sus
> > 
> > 
> > sus
> 
> sus

sus

**Comment by ghost** [comment]
> > > > > > > > > > > > sussy mogus
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > > > ?
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > > According to @aflacc, sus
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > >
> 
> > > > 
> 
> > > > 
> 
> > > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > > 
> 
> > > > 
> 
> > > > 
> 
> > > > > sus
> 
> > > > 
> 
> > > > 
> 
> > > > sus
> 
> > > 
> 
> > > 
> 
> > > sus
> 
> > 
> 
> > sus
> 
> 
> 
> sus

sus
